src/sentry/incidents/subscription_processor.py

```diff
@@ -716,11 +742,13 @@ def handle_trigger_actions(
             if incident_trigger.status == TriggerStatus.ACTIVE.value
             else AlertRuleTriggerActionMethod.RESOLVE.value
         )
-        try:
+
+        # NOTE: all incident_triggers are derived from self.active_incident
+        incident = self.active_incident
+        if not incident:
+            # NOTE: resolve incident would have cleared the active incident cache
+            # So fetch the incident if it has
             incident = Incident.objects.get(id=incident_trigger.incident_id)
```

Comment:
Why'd we remove the `try/except` here that returns? This will cause an error if we don't find the incident

Reply:
Looking at the dd charts - this metric was never actually fired 
makes sense to me - because in order to have an `IncidentTrigger` - we _must_ have an existing correlated `Incident`

Reply:
Yeah that's fair, we have a not null on the incident field on the trigger too